PublicMapping/publicmapping.github.com
# Marketing Website

# Overview
Adds a new marketing site for DistrictBuilder that replaces the splash page.

# Demo
Homepage (with Subscribe banner)
<img width="1666" alt="Screen Shot 2020-09-08 at 3 58 03 PM" src="https://user-images.githubusercontent.com/5672295/92493651-ab7ea100-f1ec-11ea-9b6d-4bee3dfe8521.png">

Feature section
<img width="1666" alt="Screen Shot 2020-09-08 at 4 02 28 PM" src="https://user-images.githubusercontent.com/5672295/92493712-bd604400-f1ec-11ea-9a05-d25029347ebf.png">

Resources section
<img width="1666" alt="Screen Shot 2020-09-08 at 4 02 35 PM" src="https://user-images.githubusercontent.com/5672295/92493726-c224f800-f1ec-11ea-9834-dd4de436db2e.png">

# Notes
- We have two articles in the screenshots, but I'll add a third if we get one in time for launch.
- Once we launch this, we'll need to let Karissa know so she can test GTM.

# Testing
- `git pull`
- Confirm that you are dropped into the page in the screenshots as opposed to the splash page
- Test all links and buttons
- Go to http://localhost:4000/wrong to see the 404 page
- Check the Contact page
- Cross-browser test
